shared/devices/container.tsx

```diff
@@ -106,14 +115,4 @@ const NamedConnected = Container.namedConnect(
 const SafeSub = Container.safeSubmitPerMount(['onBack'])
 const Connected = NamedConnected(SafeSub(ReloadableDevices))
 
-if (!Container.isMobile) {
-  // @ts-ignore fix this
-  Connected.navigationOptions = {
```

Comment:
connect actually hoists this correctly so we can apply it to the wrapped class instead of adding it to the connect after the fact